meetings/2025-04-09.md

```diff
@@ -0,0 +1,120 @@
+* MdJ: In the future when we make a spec change, we can add a note worning that WG can change it down the road.
+* JZ: So far there has been a very good relation between Practicioners and CG. When we have outreach, we should either do it with Practicioners or joint meeting. For example catalogue, one part would be community focued and another part like shapes fits more CG. We should work on that jointly instead of discussing outreach issues just here in the CG. For example having an Awesome list would be more of Practicioners topic. We will do our part in keeping CG involved on how we are progressing. I would appreciate if CG keeps Practiciones involved when it comes to outreach and community involvement.
+
+### FedCM 
+
+* Th: FedCM is on its way, but the IdP registration feature ( which could strongly improve Solid UX ) is still not part of core. I would like to share the current situation, and gather advice and feedback ( from people you have experience with the W3C process ) on how we can build a stronger case to make this feature part of FedCM core. 
+* Th: I'm not used to W3C. Idp registration feature, which is needed for Solid is not part of FedCM and still working on. I don't know how to proceed with google developers which are leading the spec.I don't want to implement it if there is no usage. I don't know if it is only Google's thing to be in that position to listen of what peole want. This feature doesn't have direct interest from google since they have strong position in IdP world. I'm bit lost in all that. 
+* eP: Thanks for bringing it up. I'll try to attend the all-day hybrid Fed-id CG meeting on Friday. I've seen your demo. Do you think it's ready to deploy it and ask other apps to also adopt it, so we have some implementation? Some signalling/endorsement from LWS WG would also be nice. Jesse also wanted to integrate with the Fediverse, maybe that could be another possibility. And I still need to understand the 'lightweight' FedCM. My understanding was that it also has IDP Registration functionality? I'll have a call with NLNet on Friday. I could help implement the lightweight one, maybe.
+* Th: IdP Registration could benefit from the ecosystem. In univerisities there are a lot of systems that can benefit from the feature. If you want to coordinate and make people aware of that.
+* Th: FedCM can be implemented in production. You need to change cookie policies which would make solidcommunity insecure. 
+* MdJ: Did you talk about that with Aaron Parecki and Sam Goto?
+* Th: Someone found a workaround with two cookie. This would require a lot of work in CSS.
+* MdJ: Are they aware of your work? When I raise the issue they just point to that workaround with a second cookie. It is only for IdP Registration which is not part of the core. https://github.com/w3c-fedid/FedCM/issues/587
```

Comment:
```suggestion
* Th: Someone found a workaround with two cookies. This would require a lot of work in CSS.
* MdJ: Are they aware of your work? When I raise the issue, they just point to that workaround with a second cookie. It is only for IdP Registration, which is not part of the core. https://github.com/w3c-fedid/FedCM/issues/587
```